wagtail/admin/tests/tests.py

```diff
@@ -373,6 +373,21 @@ def test_tags_autocomplete_custom_model(self):
         data = json.loads(response.content.decode("utf-8"))
 
         self.assertEqual(data, [])
+    def test_tags_autocomplete_limit(self):
+       
+        for i in range(15):
+            Tag.objects.create(name=f"Tag {i}", slug=f"tag-{i}")
```

Comment:
Can we use `bulk_create` here to speed it up, e.g.

```suggestion
        tags = [Tag(name=f"Tag {i}", slug=f"tag-{i}") for i in range(15)]
        Tag.objects.bulk_create(tags)
```

Reply:
Makes sense